Cards/Effects/EffectData.gd

```diff
@@ -4,19 +4,18 @@ class_name EffectData extends Resource
 
 @export var effect: EffectBase = null
 var caster: Entity = null
-var list_targets: Array[Entity] = []
 @export var value: int = 0
-@export var target_type : Enums.TargetType = Enums.TargetType.ALL_TARGETS
+@export var targeting_function: TargetingBase = null
 
-func _init(_effect: EffectBase = null, _caster: Entity = null, _list_targets: Array[Entity] = [], _value: int = 0) -> void:
+func _init(	_effect: EffectBase = null,
```

Comment:
Extra whitespace here on the `_effect` argument

Reply:
it was to have everything line up i think